.github/workflows/ci.yaml

```diff
@@ -13,6 +13,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest]
         go-version: ["1.18", "1.19", "1.20", "1.21", "1.22"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
please split the python CI into a separate job, it should be run in parallel with the Go CI

See https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-jobs-in-a-workflow